themes/cypress/layout/page.swig

```diff
@@ -21,7 +21,7 @@
           <aside id="article-toc" role="navigation">
             <div id="article-toc-inner">
               <strong class="sidebar-title">{{ __('page.contents') }}</strong>
-              {{ toc(page.content, {list_number: false}) }}
+              {{ toc(page.content, {list_number: false, max_depth: 2}) }}
```

Comment:
By specifying a maximum depth value of `2`&mdash;where the `2` refers to the maximum header level&mdash;we can control what gets generated into the TOC.  In so doing we get the desired behaviour.

https://hexo.io/docs/helpers.html#toc